### ISSUE TITLE
Makefiles as shell scripts

### ISSUE THREAD

**Issue by vhgn** [artlaman/chalice-icon-theme#5]
**Is your feature request related to a problem? Please describe.**
I used colorful icons to quickly find my `makefile` or `readme`. In this case `readme` is detectable, but I lose my `makefile` visually, as it has the same icon as `license`, `gitignore` etc.

**Describe the solution you'd like**
I would like the `makefile` to be included in shell script's category. It is basically a script, so there's a thin line there, but it will solve the problem.

**Describe alternatives you've considered**
I thought of dedicating a new icon to it, but it is completely the opposite of this project's idea.

**Additional context**
None.

Thanks for your time.

**Comment by vhgn** [comment]
> I would like the `makefile` to be included in shell script's category.

I meant it should have `>_` icon (I assumed it's an icon for shell scripts).

**Comment by artlaman** [comment]
Thanks for pointing this out 😊
I’ll take a look tomorrow.

**Comment by vhgn** [comment]
I think adding `markdown` entry to `chalice-icon-theme.json` file with icon identifier of `console` will do.

```diff
@@ -195,6 +195,7 @@
  "bat": "console.inverse",
  "cmd": "console.inverse",
  "ps1": "console.inverse",
+ "makefile": "console.inverse",
  "jpeg": "image.inverse",
  "jpg": "image.inverse",
  "gif": "image.inverse",
@@ -288,6 +289,7 @@
  "bat": "console.normal",
  "cmd": "console.normal",
  "ps1": "console.normal",
+ "makefile": "console.normal",
  "jpeg": "image.normal",
  "jpg": "image.normal",
  "gif": "image.normal",
```

I am not sure, though, if it will help. I have no experience with VS Code extensions and I don't know how to test.

**Comment by artlaman** [comment]
Not really. But I know how to do it, everything is under control. 😄

But the thing is, originally, the `shell scripts` category was intended for... the shell scripts only.  🙂
Well, it can be renamed to `scripts` and everything will become a little easier. We can even add `dockerfile` to this category.
But while  `makefile`, being just a set of instructions for `make`, might be considered a `script`, there are filetypes like: `.cabal` `.edn`, which 
perform a similar function, but are not scripts. Maybe the `code` category (`/*`) fits better?

For now I guess I'll rename the 'shell scripts' category to just 'scripts' and add 'makefile' and 'dockerfile' to it, since so far it looks like the best solution.

**Comment by vhgn** [comment]
> We can even add `dockerfile` to this category.

Yes, it is a good decision.

> But while `makefile`, being just a set of instructions for `make`, might be considered a script, ...

IMO it better fits in (shell) scripts category, I just think of it as `make` *commands* and — well — commands are made in shell scripting languages.

**Comment by vhgn** [comment]
> IMO it better fits in (shell) scripts category, ...

I meant you can leave the group name as is.

---

And it would be better for `makefile` to have shell icon (`>_`) as it is usually placed among source files. Otherwise, the icons would be the same, if you choose code icon (`/*`) for it.

**Comment by artlaman** [comment]
> And it would be better for `makefile` to have shell icon (`>_`) as it is usually placed among source files. Otherwise, the icons would be the same, if you choose code icon (`/*`) for it.
Good point. 🙂 Thank you

**Comment by artlaman** [comment]
Once again, thanks for collaboration @vhgn! 🥳 
New version will be available in the marketplace soon.

Commits 77bbf7222f051e81e84daa1dd5934e127dc1edf2 035a4485f614dd1b77a3adcd7ad9a356cc503d04 resolve #5

**Comment by artlaman** [comment]
Commits 77bbf7222f051e81e84daa1dd5934e127dc1edf2 035a4485f614dd1b77a3adcd7ad9a356cc503d04 close #5

**Comment by artlaman** [comment]
Commit 77bbf7222f051e81e84daa1dd5934e127dc1edf2 closes #5

**Comment by vhgn** [comment]
Thanks for the attention and your time!
Good luck with your projects.
